pkg/dicomio/reader.go

```diff
@@ -67,7 +67,12 @@ type Reader interface {
 	// SetCodingSystem sets the charset.CodingSystem to be used when ReadString
 	// is called.
 	SetCodingSystem(cs charset.CodingSystem)
+	// ByteOrder returns the byte order of the reader (endianness)
 	ByteOrder() binary.ByteOrder
+	// SkipPixelData returns whether or not the reader will skip pixel data
+	SkipPixelData() bool
+	// SetSkipPixelData sets whether or not the reader will skip pixel data
+	SetSkipPixelData(bool)
```

Comment:
Can we omit this setter and just add this as an option to NewReader? Does it ever need to change after we init the reader?

Reply:
> Can we [...] just add this as an option to NewReader?

Are you suggesting another arg? Would it be optional to maintain backwards-compatibility?

Reply:
See my comment above, largely I don't think this belongs here in the first place. However, if we do keep it in the reader then I don't think we need a setter. We should not worry about backwards compatibility in this package (in /pkg) because it is intended for internal use. It will not go live until we tag a new version with this, and such a change will be included in the changelog. We can further consider using the newer go modules internal/ package namspacing to help with enforcing internal-only use, but by keeping it in pkg it allows those who _really_ want to use it to have access without the compiler complaining. We can clarify this in the package comment as well.

Reply:
Perhaps I should've better clarified my question. I was asking if you're suggesting another arg I'm the NewReader() constructor in place of the setter. My concern with that was just the backwards incompat change to the constructor, but if you don't mind that, I'll make that change.

Reply:
Yep, understood, though I think my point in the thread above is I'm not sure if this arg should exist in the dicomio.Reader at all (thus probably doesn't need to be in NewReader either). Please let me know if I'm missing anything. What do you think?